doc-content/drools-docs/src/main/asciidoc/DMN/dmn-gs-creating-custom-datatypes-proc.adoc

```diff
@@ -0,0 +1,103 @@
+[id='dmn-gs-creating-custom-datatypes-proc']
+= Creating the traffic violations DMN custom data types
+
+Use the DMN designer's *Data Types* tab in {CENTRAL} to define the custom data types for the *Traffic Violations* projects.
+
+.The custom data types tab
+image::dmn/dmn-custom-datatypes-tab.png[]
+
+Alternately, you can also access the *Data Types* tab from the *Diagram properties* tab on the right. From the *Diagram properties* tab, click *Manage* from *Output data types* under *Information item*.
+
+.Diagram properties tab
+image::dmn/dmn-diagram-properties-tab.png[]
+
+The following tables show the `Violation`, `Driver`, and the `Fine` custom data types that you will create for this project.
+
+.Driver custom data type
+[cols="1,1", options="header"]
+|===
+|ID |Type
+
+|tDriver |structure
+|Name |string
+|Age |number
+|State |string
+|City |string
+|Points |number
+|===
+
+.Violation custom data type
+[cols="1,1", options="header"]
+|===
+|ID |Type
+
+|tViolation |structure
+|Code |string
+|Date |date
+|Type |string
+|Speed Limit |number
+|Actual Speed |number
+|===
+
+.Fine custom data type
+[cols="1,1", options="header"]
+|===
+|ID |Type
+
+|tFine |structure
+|Amount |number
+|Points |number
+|===
+
+.Prerequisite
+* You have created the traffic-violations DMN decision requirements diagrams (DRDs) in {CENTRAL}.
+
+.Procedure
+. From the *Data Types* tab, click *Add*.
+. First, create the *tDriver* custom data type. Enter `tDriver` as *Name* and select `structure` as the *Type*. Click *Save* to save the data type.
++
+.The tDriver custom data type
+image::dmn/dmn-tDriver-custom-datatype.png[]
+. Next to the *tDriver* data type, click the three vertical dots and select *Insert nested field*.
+.. Enter `Name` in the *Name* field and select `string` as the *Type*. Click *Save*.
+.. Click the settings icon next to the *Name* field and select *Insert field below*.
+.. Enter `Age` in the *Name* field and select `number` as the *Type*. Click *Save*.
+.. Click the settings icon next to the *Age* field and select *Insert field below*.
+.. Enter `State` in the *Name* field and select `string` as the *Type*. Click *Save*.
+.. Click the settings icon next to the *State* field and select *Insert field below*.
+.. Enter `City` in the *Name* field and select `string` as the *Type*. Click *Save*.
+.. Click the settings icon next to the *City* field and select *Insert field below*.
+.. Enter `Points` in the *Name* field and select `number` as the *Type*. Click *Save*.
+. Next, create the *tViolation* data type. From the *Data Types* tab, click *Add*.
++
+.The tViolation custom data type
+image::dmn/dmn-tViolation-custom-datatype.png[]
+. Enter `tViolation` as *Name* and select `structure` as the *Type*. Click *Save* to save the data type.
+. Next to the *tViolation* data type, click the three vertical dots and select *Insert nested field*.
```

Comment:
Changed as suggested.

Reply:
I suggested "Click the check icon [image] to save" instead of just "Click [image] to save" on purpose. It's an accessibility consideration to avoid using icons or screenshots as the only method for describing something, in case the image doesn't render or whatever. Images should always supplement the instruction but never be the sole source of it. Also for this reason we want to avoid saying things like, "Fill in the details as shown in the screenshot" or whatever. You don't do that, just saying as an example. We instead would explicitly document the items and then show them in the screenshot as a second, in-context look. You'd done that quite well in this doc, actually. Anyway, just a side note.

Reply:
@sterobin, understood your point. Changed as suggested.